config/200-clusterroles.yaml

```diff
@@ -12,6 +12,86 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: triggermesh-controller-watcher
```

Comment:
the controller MUST HAVE clusterscope for these

Reply:
Are you sure about this role? I'm kind of surprised by the set of resources - deployments, but not kservices, sources, but not transformations, etc.

Reply:
> the triggermesh-controller-watcher has all the clusterscope permissions the controller was complaining about, so at the moment these are the once that ARE NEEDED for the controller to function. It's possible some more maybe required, but there are the ones i've encountered in my testing. Do you think it would make more sense to give list and watch permissions for everything that's listed under the triggermesh-controller clusterrole?

It's always nice to be sure about our own product and changes that we make. RBAC that's created based on the experiment scares me a little.
@odacremolbap maybe you have some ideas why this role includes these resources only?